src/app/rmm/plugin/tinymce.plugin.ts

```diff
@@ -0,0 +1,68 @@
+// --------- BEGIN RUNBOX LICENSE ---------
+// Copyright (C) 2016-2018 Runbox Solutions AS (runbox.com).
+// 
+// This file is part of Runbox 7.
+// 
+// Runbox 7 is free software: You can redistribute it and/or modify it
+// under the terms of the GNU General Public License as published by the
+// Free Software Foundation, either version 3 of the License, or (at your
+// option) any later version.
+// 
+// Runbox 7 is distributed in the hope that it will be useful, but
+// WITHOUT ANY WARRANTY; without even the implied warranty of
+// MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the GNU
+// General Public License for more details.
+// 
+// You should have received a copy of the GNU General Public License
+// along with Runbox 7. If not, see <https://www.gnu.org/licenses/>.
+// ---------- END RUNBOX LICENSE ----------
+import { timeout, share } from 'rxjs/operators';
+import { Observable } from 'rxjs';
+import { RMM } from '../../rmm';
+
+declare const tinymce: any;
+
+export class TinyMCEPlugin {
+    constructor(
+        public app: RMM,
```

Comment:
This looks unused here – and I assume it is, since there's no reason why the editor should need an RMMapi :­P

Since this is copypasted from compose.component.ts, can you make ComposeComponent use TinyMCEPlugin so that we don't have duplicate code?

Reply:
fixed

Reply:
Cool, thanks.

Reply:
Perfect! Can we now have the same code reused in ComposeComponent so we don't duplicate things?

Reply:
Wonderful :)